setup.cfg

```diff
@@ -18,7 +18,10 @@
 
 [aliases]
 # so that running python setup.py test invokes pytest
-test=pytest
+test=
+    pytest
+    flake8
+    flake8-type-checking; python_version > "3.7"
```

Comment:
This isn't actually adding `flake8-type-checking` as a dependency. In Isodatetime we still haven't switched from `setup.py` to `setup.cfg` properly. And in fact it would make more sense to switch to `pyproject.toml` at this point